### PR TITLE
Test Python 3.7 in Travis

### PR DESCRIPTION
**Pull request by warenlg**
Signed-off-by: Waren Long <waren@sourced.tech>

Issue : https://github.com/src-d/modelforge/issues/46

### CODE REVIEW

**Comment by warenlg** [comment]
`pip install numpy` fails in Travis Python 3.7 environment, related SO https://stackoverflow.com/questions/48051779/pip-install-scipy-failed-in-travis-ci-python-3-7-env

Travis Python 3.7 passes for https://github.com/src-d/style-analyzer because neither `numpy` nor `scipy` are in the requirements

**Comment by vmarkovtsev** [comment]
numpy is installed in style-analyzer. Proof: https://travis-ci.org/src-d/style-analyzer/jobs/419743055#L703

**Comment by warenlg** [comment]
Yes, looks good now

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 asdf==2.0.1
 lz4==2.0.2
-numpy==1.14.0
-scipy==1.0.0
+numpy>=1.14
```
Comment (vmarkovtsev) [inline requirements.txt]: Nope we must pin the exact version in requirements.

You can change in `.travis.yml`:

**before**
```
pip install -r requirements.txt codecov
```

**after**
```
pip install codecov
```

thus the deps will be loaded from `setup.py` on the next step and `setup.py` is very liberal (does not target 100% reproducability).

Comment (warenlg) [reply requirements.txt]: Ok I see now, thanks @vmarkovtsev

**Pull request merged**